java/java.source.base/src/org/netbeans/api/java/source/TreeMaker.java

```diff
@@ -267,6 +267,18 @@ public CaseTree Case(List<? extends ExpressionTree> patterns, Tree body) {
         return delegate.Case(patterns, body);
     }
     
+    /**
+     * Creates a new CaseTree for a rule case (case &lt;constants&gt; -> &lt;body&gt;).
+     *
+     * @param patterns the labels for this case statement.
+     * @param body the case's body
+     * @see com.sun.source.tree.CaseTree
+     * @since 2.39
+     */
+    public CaseTree CaseMultipleSwitchPatterns(List<? extends Tree> patterns, Tree body) {
```

Comment:
Maybe just `CaseMultiplePatterns`? Just a suggestion.

Also - there should probably be the similar method taking statements instead of body.